particuleio/terraform-kubernetes-addons
# feat: add rabbitmq-cluster-operator chart

# Add rabbitmq-cluster-operator chart 

## Description

- Add the rabbitmq-operator addon [link](https://www.rabbitmq.com/kubernetes/operator/operator-overview.html)
- Generate documentation for rabbitmq-operator
- Add rabbitmq-operator in aws, azure and scaleway modules

See [Bitnami's Artifact page](https://artifacthub.io/packages/helm/bitnami/rabbitmq-cluster-operator?modal=security-report)

### How to test this addon:

- [Create a RabbitMQ Cluster](https://www.rabbitmq.com/kubernetes/operator/using-operator.html#create)
- [Verify the Cluster is working properly](https://www.rabbitmq.com/kubernetes/operator/using-operator.html#verify-instance)

### Checklist

- [x] README.md has been updated after any changes to variables and outputs. See https://github.com/particuleio/terraform-kubernetes-addons/#doc-generation
